templates/common/_base/files/configure-ovs-network.yaml

```
@@ -84,6 +84,16 @@ contents:
         echo "MTU found for iface: ${iface}: ${iface_mtu}"
       fi
 
+      # store old conn for later
+      old_conn=$(nmcli --fields UUID,DEVICE conn show --active | grep ${iface} | awk '{print $1}')
```

No need to both grep and awk
```suggestion
      old_conn=$(nmcli --fields UUID,DEVICE conn show --active | awk "/${iface}/ {print \$1}")
```

---

this made me realize we probably need to be more deterministic here how we match the interface. If iface was eth1, and we had eth10 on the system, we would accidentally grab the wrong one. Going to change it to something like:
```
nmcli --fields UUID,DEVICE conn show --active | awk "/\s${iface}\s*\$/ {print \$1}"
```